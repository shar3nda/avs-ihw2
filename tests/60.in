np9&'Y));}U(=T4>~==bB$8b>/&"3,M^0gnX5sDrA7t%%'\)PySXJfu>Q`!&#4<'?.U
ni<6KzEVy>1