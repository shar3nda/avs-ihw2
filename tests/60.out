Lowercase: 16, uppercase: 17